sources/models/src/modeled_types/oci_defaults.rs

```diff
@@ -120,10 +157,155 @@ mod oci_defaults_rlimits {
 
     #[test]
     fn linux_rlimit_strings() {
-        check_rlimit_strings(
-            OciDefaultsResourceLimitType::MaxOpenFiles,
-            "max-open-files",
-            "RLIMIT_NOFILE",
-        );
+        let rlimits = [
+            (MaxAddressSpace, "max-address-space", "RLIMIT_AS"),
+            (MaxCoreFileSize, "max-core-file-size", "RLIMIT_CORE"),
+            (MaxCpuTime, "max-cpu-time", "RLIMIT_CPU"),
+            (MaxDataSize, "max-data-size", "RLIMIT_DATA"),
+            (MaxFileLocks, "max-file-locks", "RLIMIT_LOCKS"),
+            (MaxFileSize, "max-file-size", "RLIMIT_FSIZE"),
+            (MaxLockedMemory, "max-locked-memory", "RLIMIT_MEMLOCK"),
+            (MaxMsgqueueSize, "max-msgqueue-size", "RLIMIT_MSGQUEUE"),
+            (MaxNicePriority, "max-nice-priority", "RLIMIT_NICE"),
+            (MaxOpenFiles, "max-open-files", "RLIMIT_NOFILE"),
+            (
+                MaxPendingSignals,
+                "max-pending-signals",
+                "RLIMIT_SIGPENDING",
+            ),
+            (MaxProcesses, "max-processes", "RLIMIT_NPROC"),
+            (
+                MaxRealtimePriority,
+                "max-realtime-priority",
+                "RLIMIT_RTPRIO",
+            ),
+            (MaxRealtimeTimeout, "max-realtime-timeout", "RLIMIT_RTTIME"),
+            (MaxResidentSet, "max-resident-set", "RLIMIT_RSS"),
+            (MaxStackSize, "max-stack-size", "RLIMIT_STACK"),
+        ];
+
+        for rlimit in rlimits {
+            check_rlimit_strings(rlimit);
+        }
+    }
+}
+
+/// OciDefaultsResourceLimitValue specifies what all values are
+/// allowed to be set for rlimit in default spec.
```

Comment:
We should call out `unlimited` and how this special case needs to have something more than a `u32`.